web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -209,35 +230,62 @@ class CVEAudit extends React.Component<Props, State> {
               onChange={this.onCVEChange}
             />
           </div>
-          <div>
-            {ALL.map((status) => {
-              return (
-                <div className="checkbox">
-                  <label>
-                    <input
-                      type="checkbox"
-                      checked={this.state.statuses.includes(status)}
-                      onChange={(e) => {
-                        if (this.state.statuses.includes(status)) {
-                          this.setState({
-                            statuses: this.state.statuses.filter((x) => x !== status),
-                          });
-                        } else {
-                          this.setState({
-                            statuses: this.state.statuses.concat([status]),
-                          });
-                        }
-                      }}
-                    />
-                    <i
-                      className={"fa fa-big " + PATCH_STATUS_LABEL[status].className}
-                      title={PATCH_STATUS_LABEL[status].label}
-                    />
-                    <span>{" " + PATCH_STATUS_LABEL[status].label}</span>
-                  </label>
-                </div>
-              );
-            })}
+          <div className="" style={{ maxWidth: "770px", marginTop: "16px" }}>
+            <div className="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
While the accordion pattern can be useful in many situations, I don't think this specific one works well. The core problem is that traditionally, clicking on a label element should toggle the relevant input, in this case the checkbox. Here it instead toggles the accordion, so it doesn't match user expectations.

I would prefer to move the descriptions into tooltips since they're pretty short and then show exactly as the user interacts with the element, that way we can also keep the default expected behavior of the label click modifying the checkbox. What do you think @parlt91?

Reply:
I agree with what you mentioned that we should stick with the behavior that clicking on the text ticks the checkboxes. We use tooltips to display descriptions all over the UI so that doesn't sound like a bad solution to me either.

Reply:
@Etheryte @parlt91 Thanks for the review and suggestions. I removed the accordion and replaced it with showing a tooltip message per your recommendations, but I thought it would be better to display the tooltip message for each client. Although the messages are static I think it would be more convenient for users to hover over the patch status on one client to learn more about it instead of scrolling up and down.

![image](https://github.com/uyuni-project/uyuni/assets/21198231/348e9bae-d085-4087-bf89-ac370ad8f658)

Reply:
I have a question: any idea how to replace Uyuni with SUSE Manager when the SUSE Manager theme is used?

Reply:
Thanks, I rephrased the tooltip messages to not mention Uyuni.